packages/ui-kit/src/components/index.ts

```diff
@@ -1,12 +1,13 @@
+export * from './AccessTokenProvider'
+export * from './Autocomplete'
```

Comment:
Recommend to hide this from our public API for now.

Reply:
Yeah, I agree. Let's hide it for now.